### ISSUE TITLE
Consider rust wayland-sys

### ISSUE THREAD

**Issue by loops-and-spells** [deno-windowing/deno_wayland#1]
Looks like the .so files don't export the static methods. Using a wrapper library may make this easier?
https://crates.io/crates/wayland-sys

**Comment by load1n9** [comment]
looking into it since a big thing is i want to avoid any bridge code between the libraries and deno but no idea if its possible with wayland

**Comment by loops-and-spells** [comment]
I don't think we can get access to the static methods without wrapping them and rebuilding an .so
I'm not sure how complicated that could get without introducing side-effects. 

Wonder if automating adding deno bindgen to the crate would be possible.
https://github.com/denoland/deno_bindgen
